docs/content/envoy-introduction.md

```diff
@@ -49,6 +49,8 @@ The OPA-Envoy plugin supports the following configuration fields:
 | `plugins["envoy_ext_authz_grpc"].proto-descriptor`  | No       | Set the path to a pb that enables the capability to decode the raw body to the parsed body. Default: turns this capability off.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `plugins["envoy_ext_authz_grpc"].grpc-max-recv-msg-size` | No       | Set the max message size in bytes the GRPC server can receive. Defaults to 4MB.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| `plugins["envoy_ext_authz_grpc"].grpc-max-send-msg-size` | No       | Set the max message size in bytes the GRPC server can send. Defaults to `math.MaxInt32`.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
```

Comment:
[nit] I'm not sure everyone knows roughly what `math.MaxInt32` means in this context. Can we spell it out, or give some other indication, please?

Reply:
Sure, I've replaced it with a more explicit description.